content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -68,9 +68,13 @@ Follow these instructions to prepare a kind cluster for Istio installation.
     Deleting cluster "istio-testing" ...
     {{< /text >}}
 
+## Setup MetalLB for kind
+
+kind does not have any built-in way to provide IP addresses to your `Loadbalancer` service types, to ensure IP address assignments to `Gateway` Services please consult [this guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for more information.
+
 ## Setup Dashboard UI for kind
 
-kind does not have a built in Dashboard UI like minikube. But you can still setup Dashboard, a web based Kubernetes UI, to view your cluster.
+kind does not have a built-in Dashboard UI like minikube. But you can still setup Dashboard, a web based Kubernetes UI, to view your cluster.
```

Comment:
```suggestion
kind does not have a built-in Dashboard UI like minikube. But you can still setup Dashboard, a web-based Kubernetes UI, to view your cluster.
```

Reply:
Thanks I also managed to fix the ordering I mentioned earlier. Let me push this too! thank you